test/e2e/resize_nodes.go

```diff
@@ -38,38 +39,62 @@ import (
 const serveHostnameImage = "gcr.io/google_containers/serve_hostname:1.1"
 
 func resizeGroup(size int) error {
-	// TODO: make this hit the compute API directly instread of shelling out to gcloud.
-	output, err := exec.Command("gcloud", "preview", "managed-instance-groups", "--project="+testContext.CloudConfig.ProjectID, "--zone="+testContext.CloudConfig.Zone,
-		"resize", testContext.CloudConfig.NodeInstanceGroup, fmt.Sprintf("--new-size=%v", size)).CombinedOutput()
-	if err != nil {
-		Logf("Failed to resize node instance group: %v", string(output))
+	if testContext.Provider == "gce" {
```

Comment:
(Oh, this isn't the PR that is mysteriously failing.  That one is my much simpler SSH-user PR.  Never mind!)